src/app/public/modules/autocomplete/autocomplete.component.scss

```diff
@@ -11,4 +11,8 @@
     right: 0;
     bottom: 0;
   }
+
+  .sky-autocomplete-no-results-found {
+    padding: 3px 20px
```

Comment:
We have SCSS variables for these values; try:
```
padding: $sky-space-xxs $sky-space-xl;
```

Reply:
Cool, updated to use those values instead.